tests/TestHelpers/GraphHelper.php

```diff
@@ -2293,4 +2293,30 @@ public static function getPermissionRoleDefinition(
 			self::getRequestHeaders()
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $requestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $fileId
+	 *
+	 * @return ResponseInterface
+	 */
+	public static function getActivities(
+		string $baseUrl,
+		string $requestId,
+		string $user,
+		string $password,
+		string $fileId
+	): ResponseInterface {
+		$fullUrl = $baseUrl . "/graph/v1beta1/extensions/org.libregraph/activities?kql=itemid%3A" . $fileId;
```

Comment:
is `fileId` a must param?

Reply:
looks like it is required for now: https://github.com/owncloud/ocis/issues/9194

Reply:
without `fileId` we don't get the response, so it can't be optional.

Reply:
Could put a comment here?

Reply:
of course